### ISSUE TITLE
SRM treats windows shortcuts (.lnk) differently than steam & causes lots of issues

### ISSUE THREAD

**Issue by klepp0906** [SteamGridDB/steam-rom-manager#155]
This is tough to explain so im going to attach an image.

The TLDR is at current you cant use SRM to add non-steam windows games to steam.   I put all my non-steam games into a games folder, like many do im sure.  Its populated with installer created shortcuts pointing to the executable buried wherever it was installed.  

I created a parser, and it added the shortcuts no problem.   At first glance everything was fine.  The problem began when i decided to play a game that was added via that fashion.

Game would launch no problem.   However steam didnt treat it as such, and no controls were passed.   Games illustrated the fact with either "no controller connected" or tooltips that would normally say "Press A" would say "Press ENTER" or what-have-you. 

If I tried to play remotely via steamlink, games that normally worked would stream to a black screen instead.

Ultimately the crux of the issue is what you see in the following image, with the left side being added by SRM, and the right being added manually by pointing to the same exact shortcut via steams "add non-steam game" button. 

It seems steam goes ahead and does the translation for you, following the shortcut and creating the steam shortcut same as if you had pointed to the executable itself.   SRM on the other hand, creates it as the shortcut, which leaves you control-less and often stream-less (some games were able to stream that way, but still couldnt control them). 

So what im asking is for is a means for the parser to translate and create the steam shortcuts based on where the windows (.lnk) shortcuts point, just like steam.   Otherwise, your SoL using SRM for anything other than roms as windows games have exes buried to differing depths, and some have multiple etc.   No other way to do it than with shortcuts. 

Hoping this is an easy fix. 

![Shortcuts](https://user-images.githubusercontent.com/10392920/71740434-302d4a00-2e2a-11ea-9bf7-4b5f998b04de.png)
